src/actions/console.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const alt = require('../alt');
+
+class ConsoleActions {
+  clearOutput() {
+    this.dispatch();
+  }
+
+  output(evt) {
```

Comment:
i don't like calling this `evt`, as typically that is reserved for UI events and isn't descriptive in this context